### PR TITLE
Visible scanlines

### PR DESCRIPTION
**Pull request by johnor**

### CODE REVIEW

File: core/test/src/test_ppu.cpp [hunk 1]
```diff
@@ -360,6 +362,131 @@ TEST_F(PpuTest, increment_vram_addr_by_32_after_reading) {
...
+
+    // Clear scrolling
+    ppu->write_byte(0x2005, 0);
+    ppu->write_byte(0x2005, 0);
+
+    // Nametable on cycle 1 set to index 2, and index 3 on cycle 9
+    EXPECT_CALL(mmu, read_byte(0x2000)).WillOnce(Return(0x02));
+    EXPECT_CALL(mmu, read_byte(0x2001)).WillOnce(Return(0x03));
+
+    // Attribyte on cycle 3 and 12.
```
Comment (robinlinden) [inline core/test/src/test_ppu.cpp:381]: Attribyte -> Attribute?

File: core/test/src/test_ppu.cpp [hunk 2]
```diff
@@ -360,6 +362,131 @@ TEST_F(PpuTest, increment_vram_addr_by_32_after_reading) {
     EXPECT_EQ(expected, registers);
 }
 
+TEST_F(PpuTest, visible_two_sub_cycles) {
```
Comment (robinlinden) [inline core/test/src/test_ppu.cpp:365]: The order would be clearer if this was tested with InSequence.

File: core/test/src/test_ppu.cpp [hunk 3]
```diff
@@ -360,6 +362,131 @@ TEST_F(PpuTest, increment_vram_addr_by_32_after_reading) {
...
+    EXPECT_CALL(mmu, read_byte(0x03 * 16u + 8u)).WillOnce(Return(0x99));
+
+    for (int i = 0; i < 17; ++i) {
+        ppu->execute();
+    }
+
+    EXPECT_EQ(expected, registers);
+}
+
+TEST_F(PpuTest, visible_scaneline) {
```
Comment (robinlinden) [inline core/test/src/test_ppu.cpp:404]: scaneline -> scanline?

**Review by robinlinden — APPROVED** [review]

File: core/test/src/test_ppu.cpp (hunk 2 above)
Comment (johnor) [reply core/test/src/test_ppu.cpp:367]: I left the test for the whole line as is though.

**Review by robinlinden — APPROVED** [review]

**Pull request merged**